<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sound of Maple</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Resources/replace.png</normaloff>:/Resources/replace.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	border:none;
}

QPushButton:hover
{
	background-color: rgb(0, 85, 255);
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_openFile">
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Resources/replace.png);</string>
      </property>
      <property name="text">
       <string>choose</string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_last">
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Resources/replace.png);</string>
      </property>
      <property name="text">
       <string>上一首</string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_switch">
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Resources/replace.png);</string>
      </property>
      <property name="text">
       <string>播放</string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_next">
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Resources/replace.png);</string>
      </property>
      <property name="text">
       <string>下一首</string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>QPointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Resources/replace.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="songListWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>311</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文楷体&quot;;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>381</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_cTime">
      <property name="text">
       <string>00：00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="playProcessSlider">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_totalTime">
      <property name="text">
       <string>00：00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>42</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_sound_1">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Resources/replace.png);</string>
      </property>
      <property name="text">
       <string>音量</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_sound_2">
      <property name="text">
       <string>50</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="soundSlider">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>281</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>281</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
